packages/react-cli/src/commands/fetch-configs.ts

```diff
@@ -0,0 +1,29 @@
+import { Command } from '@oclif/command';
+import { ensureDependencies } from '../tasks/ensure-dependiencies';
+import Listr from 'listr';
+import { downloadTSConfigFiles } from '../tasks/download-ts-config';
+import { exec } from '../utils/shell';
+
+export class FetchConfigsCommand extends Command {
+  static description = 'Fetches config files for linting and typescript projects';
+
+  static examples = ['$ react fetch-configs'];
+
+  async run() {
+    await ensureDependencies();
+
+    const currentWorkingDirectory = './';
```

Comment:
now I am wondering if this should be `__dirname`

Reply:
hmm. a better solution may be to go a step further and assert that the directory has a package.json file